src/utilities/stopWatch.js

```diff
@@ -0,0 +1,78 @@
+export class Stopwatch {
+  constructor() {
+    this.buttonStart = document.querySelector('.button-start');
+    this.buttonStop = document.querySelector('.button-stop');
+    this.stopWatchScreen = document.querySelector('.stop-watch-screen');
```

Comment:
If there is only one button of a certain type and you want to use it through JavaScript, use IDs instead of classes in the `querySelector`. Feel free to use those classes for styling, though.